js/app.js

```diff
@@ -1 +1,80 @@
-// Your code here
+// Model
+
+$(document).ready(function() {
+
+var model = {
+      loggedIn: false,
+      userId: '',
+      username: '',
+      category: [],
+      articles: []
+  };
+
+// View
+
+function renderUser() {
+  var templateSource = $('#user-template').html();
+  var template = Handlebars.compile(templateSource);
+  var templateRender = template(model);
+  $('#loginContainer').html(templateRender);
+};
+
+renderUser();
```

Comment:
You need to be careful calling this here. Remember that the compiling the templates (`Handlebars.compile`) usually requires the document to be ready. Since this function call is happening not in a `$(document).ready` listener, it probably will fail.